swap/src/seed.rs

```diff
@@ -1,6 +1,14 @@
+use crate::fs::ensure_directory_exists;
 use ::bitcoin::secp256k1::{self, constants::SECRET_KEY_SIZE, SecretKey};
+use pem::{encode, Pem};
 use rand::prelude::*;
-use std::fmt;
+use std::{
+    ffi::OsStr,
+    fmt,
+    fs::{self, File},
+    io::{self, Write},
+    path::{Path, PathBuf},
+};
```

Comment:
Looking at the dependencies and the functionality, this module could be a candidate to be its own crate (for now, just in the workspace) but maybe later on also on crates.io.

- We've basically imported that thing from comit-rs and it hasn't changed much since then
- Minimal dependencies
- Provides a clear value

Reply:
`seedfile` is available on crates.io :)

Reply:
Good point.

Reply:
> `seedfile` is available on crates.io :)

Can't find it...